packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -202,10 +202,16 @@ export namespace CommonCommands {
         label: 'About'
     };
 
-    export const OPEN_PREFERENCES: Command = {
-        id: 'preferences:open',
```

Comment:
command ids are apis, renaming them is a breaking change, please revert

Why do we need 2 btw?

Reply:
Our first thought was that the two preferences widgets would coexist somehow, so we made two different commands for the different views. Should I focus on adding the JSON view as a component in the UI view or should I just not include the JSON view?